### DISCUSSION TITLE
Changing Unit of Iota

### DISCUSSION THREAD

**Discussion by alorraine** [iotaledger/firefly#651] [Features / Enhancements]
I wanted to send 11 Ki, before I knew about the 1Mi limit.

Without me really realising, I sent 11Mi. This is because I entered "11" into the box, then changed the unit. This did change the unit and the amount to 11000, but it didn't quite register with me because I had already entered 11.

Should the absolute number entered stay the same when changing units?

Also though there might be a double check screen, not just a password request. "Are you sure you want to send 11000 Ki (11Mi) to atoi......" or something like that?

**Comment by maxwellmattryan** [comment]
Hey @alorraine, sorry that that had happened. Thinking about it, if a user intends to switch the IOTA denomination then it's more likely that they mean to select that same number in whatever unit rather than send that same value in a different denomination (I would think, but could be wrong)... so I'm for switching it to staying the same when changing units. @cvarley100 what're your thoughts?

**Reply by rafaeldjpbrochado** [reply]
Agreed.
